lib/tasks/bundlers/generateLibraryPreload.js

```diff
@@ -31,11 +31,63 @@ function getDefaultLibraryPreloadFilters(namespace, excludes) {
 	return filters;
 }
 
-function getBundleDefinition(namespace, excludes) {
+function getBundleDefinition(namespace, excludes, preloadMode = "preload") {
```

Comment:
There should not be any changes to the current way how library preloads are created. The legacy h2-preload should not be re-introduced. First step is only to make the new bundle section mode available.